src/xcode/ENA/ENA/Source/RootCoordinator.swift

```diff
@@ -144,6 +144,7 @@ class RootCoordinator: RequiresAppDependencies {
 		diaryCoordinator.viewController.tabBarItem = diaryTabBarItem
 
 		tabBarController.tabBar.tintColor = .enaColor(for: .tint)
+		tabBarController.delegate = homeCoordinator.rootViewController
```

Comment:
Does the feature only work for the home screen?

Reply:
@ArturFriesen actually, it is implemented for all 4 tabs. The scrolling is intended only on the root view controller of the respective tab; this follows the behaviour of native apps like the Health app. 
It works only if that controller has a table view; this was tested for both a native UITableViewController and a cwa DynamicTableViewController.
I have tested it on simulator; not tested for iOS 12.5.
I have tested on all 4 tabs; however, the Certificate tab I have tested only with its initial info screen (I don't know how to add certificates with the simulator - is there a trick for this?).

Reply:
Please re-review @ArturFriesen. Thanks!

Reply:
This line was confusing. `homeCoordinator.rootViewController` is the root controller of the home tab. And it was handling the scrolling of the other tabs. From hierarchy perspective this is confusing. Only by looking at this line, it looks like only the root of home screen will be handled.

However, I like the solution with the `TabBarScrolling` a lot more! It is not confusing from hierarchy perspective and encapsulates the behavior in an own component. Thanks!